ReactQt/runtime/src/reactredboxitem.cpp

```diff
@@ -134,4 +134,8 @@ void ReactRedboxItem::updateErrorMessage(const QString& message, const QList<QVa
     d->setStack(stack);
 }
 
+QString ReactRedboxItem::errorMessage() const {
+    return d_func()->redbox->property("message").toString();
```

Comment:
String "message" has 3 occurrences in the file, it would be nice to make it constant to simplify possible changes in the future.